src/framework/core/js/Fluid.js

```diff
@@ -288,6 +312,7 @@ var fluid = fluid || fluid_3_0_0;
     /** Undo the effect of the most recent "setLogging", returning the logging system to its previous state
      * @return {LogLevel} The logLevel that was just popped
      */
+    // TODO: This function is documented, but untested
```

Comment:
We have an existing test in `FluidJSTests.js`:

https://github.com/amb26/infusion/blob/FLUID-6148/tests/framework-tests/core/js/FluidJSTests.js#L696-L709

Were you thinking we wanted something more specific for `fluid.popLogging`? Is that something we should do as part of this PR, or is just adding the `TODO` comment for the future good?

Reply:
Good catch, thanks!